kpops/cli/config.py

```diff
@@ -29,7 +29,43 @@ class TopicNameConfig(BaseSettings):
     )
 
 
-class PipelineConfig(BaseSettings):
+class SchemaRegistryConfig(BaseSettings):
+    """Configuration for Schema Registry."""
+
+    enabled: bool = Field(
+        default=False,
+        description="If the Schema Registry handler should be initialized.",
+    )
+    url: AnyHttpUrl = Field(
+        default=parse_obj_as(AnyHttpUrl, "http://localhost:8081"),
+        env=f"{ENV_PREFIX}SCHEMA_REGISTRY_URL",
+        description="Address of the Schema Registry.",
+    )
+
+
+class KafkaRestConfig(BaseSettings):
+    """Configuration for Kafka REST Proxy."""
+
+    url: AnyHttpUrl = Field(
+        # For validating URLs use parse_obj_as
+        # https://github.com/pydantic/pydantic/issues/1106
```

Comment:
good note, could you move this up to the first usage (in `SchemaRegistryConfig`)